src/api/api.ts

```diff
@@ -0,0 +1,75 @@
+import axios from 'axios';
+
+export const client = axios.create({
+  baseURL: 'https://mate.academy/students-api',
+})
+
+//#region User
+export const getUserByEmail = (email: string) => {
+  return client.get('/users', {
+    params: {
+      email,
+    }
+  })
+}
+
+export const createUser = (email: string, name: string) => {
+  return client.post('/users', {
+    email,
+    name,
+  })
+}
```

Comment:
Again, it's recommended to specify the expected return type for this function. This will help to prevent potential errors in the future.

Reply:
done